static/i18n/en/app.yml

```diff
@@ -283,24 +283,26 @@ settings:
     about: Help
   display:
     desc:
-    language: Language
-    languageDesc: Choose the language to display.
+    language: Display language
+    languageDesc: Set the language displayed in Ledger Live.
     counterValue: Base currency
-    counterValueDesc: Choose the currency to display next to your balance and operations. An exchange is used (via BTC pair).
+    counterValueDesc: Choose the currency to display next to your balance and operations.
+    exchange: Rate provider ({{ticker}})
+    exchangeDesc: Choose the provider of the base currency exchange rates
     region: Region
-    regionDesc: Choose the region in which you’re located to set the application’s time zone.
+    regionDesc: Choose the region in which you’re located to set the Ledger Live's time zone.
     stock: Regional market indicator
     stockDesc: Choose Western to display an increase in market value in blue. Choose Eastern to display an increase in market value in red.
   currencies:
-    desc: Select a cryptocurrency to edit its settings.
+    desc: Select a cryptocurrency to edit its settings
     exchange: Rate provider ({{ticker}})
-    exchangeDesc: Choose the provider of the base currency exchange rates (via BTC).
+    exchangeDesc: Choose the provider of the base currency exchange rates
     confirmationsToSpend: Number of confirmations required to spend
-    confirmationsToSpendDesc: Set the number of confirmations required for your funds to be spendable. # A higher number of confirmations decreases the probability that a transaction is rejected.
+    confirmationsToSpendDesc: Set the number of network confirmations required for your crypto assets to be spendable. # A higher number of confirmations decreases the probability that a transaction is rejected.
     confirmationsNb: Number of confirmations
-    confirmationsNbDesc: Set the number of blocks a transaction needs to be included in to consider it as confirmed. # A higher number of confirmations increases the certainty that a transaction cannot be reversed.
+    confirmationsNbDesc: Set the number of network confirmations for a transaction to be marked as confirmed. # A higher number of confirmations increases the certainty that a transaction cannot be reversed.
     transactionsFees: Default transaction fees
-    transactionsFeesDesc: Select your default transaction fees. The higher the fee, the quicker the transaction will be processed.
+    transactionsFeesDesc: Select your default transaction fees. The higher the fee, the faster the transaction will be processed.
```

Comment:
(not used? maybe old settings)

Reply:
Changed to faster for consistency.

Reply:
no but i mean, transactionsFeesDesc is not used. there is no longer such settings available. because fees are always at the time you do a transaction.

so you can remove this for now.